applications/app/model/TagIndexListingMetaData.scala

```diff
@@ -3,7 +3,7 @@ package model
 case class SubjectsListing() extends StandalonePage {
   override val metadata = MetaData.make(
     id = "index/subjects",
-    section = "Index",
+    sectionSummary = Some(SectionSummary.fromId("Index")),
```

Comment:
In `TagIndexPageMetadata` the sectionSummary string has been extracted into `private val tagType: String` - could do the same here for consistency.